openwhisk/index.md

```diff
@@ -28,11 +28,52 @@ For more details about how {{site.data.keyword.openwhisk_short}} works, see [Abo
 ## Setting up the {{site.data.keyword.openwhisk_short}} CLI
 {: #openwhisk_start_configure_cli}
 
-You can use the {{site.data.keyword.openwhisk_short}} command line interface (CLI) to set up your namespace and authorization key. 
-Go to [Configure CLI](https://new-console.{DomainName}/openwhisk/cli){: new_window} and follow the instructions to install it. 
-Note that you must have Python 2.7 installed on your system to use the CLI.
+The OpenWhisk command line interface (CLI) requires Python 2.7.
 
-After {{site.data.keyword.openwhisk_short}} is set up with the CLI, you can begin using it from the command line or through REST APIs.
+- If you cloned the OpenWhisk respository, you will find the CLI in `openwhisk/bin/wsk`.
```

Comment:
Hi @mdeuser None of your updates were merged, as we were waiting for Carlos to let us know if they should be made. In the meantime, I made a lot of editorial updates over on docs-staging, and then moved them here to docs. So if any of your updates are needed, I think they should first be made at docs-staging, I can then merge and review in a staging build, and then move to docs for a production build. Otherwise we would have collisions and a lot of possible problems and cleanup.